src/chapters/strings/string-methods.rst

```diff
@@ -0,0 +1,113 @@
+
+As we have learned, strings are immutable. Therefore, string methods will not change the value of a string itself, but instead will *return* a new string that is the result of the given operation.
+
+We saw this behavior in the ``toLowerCase`` example.
+
+.. admonition:: Example
+
+   .. sourcecode:: js
+   
+      let nonprofit = "LaunchCode";
+
+      console.log(nonprofit.toLowerCase());
+      console.log(nonprofit);
+
+   **Output**
+
+   ::
+
+      launchcode
+      LaunchCode
+
+While ``nonprofit.toLowerCase()`` evaluated to ``"launchcode"``, the value of ``nonprofit`` was left unchanged. This will be case for each of the string methods.
+
+Common String Methods
+---------------------
+
+.. index::
+   single: string; methods
+
+Here we present the most commonly-used string methods. You can find documentation for other string methods at:
+
+- `W3Schools <https://www.w3schools.com/jsref/jsref_obj_string.asp>`_
+- `MDN <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String#Methods_2>`_
+
+
+.. list-table:: Common String Methods
+   :header-rows: 1
+
+   * - Method
+     - Syntax
+     - Description
+   * - :ref:`indexOf <string-indexof-examples>`
+     - ``stringName.indexOf(character)``
+     - Returns the index of the first occurence of the character in the string, and returns -1 if the character is not found.
+   * - :ref:`toLowerCase <string-tolowercase-examples>`
+     - ``stringName.toLowerCase()``
+     - Returns a copy of the given string, with all uppercase letters converted to lowercase.
+   * - :ref:`toUpperCase <string-touppercase-examples>`
+     - ``stringName.toUpperCase()``
+     - Returns a copy of the given string, with all lowercase letters converted to uppercase.
+   * - :ref:`trim <string-trim-examples>`
+     - ``stringName.trim()``
+     - Returns a copy of the given string with a leading and trailing whitespace removed.
+   * - :ref:`replace <string-replace-examples>`
+     - ``stringName.replace(searchChar, replacementChar)``
+     - Given a search string ``searchChar`` and a replacement value ``replacementChar``, replace the first occurence of ``searchChar`` replaced by ``replacementChar``.
+   * - :ref:`slice <string-slice-examples>`
+     - ``stringName.slice(i, j)``
+     - Return the substring consisting of characters from index ``i`` through index ``j-1``.
+
+.. tip:: String methods can be combined in a process called **method chaining**. Given ``word = 'JavaScript';``, word.toUpperCase() returns ``JAVASCRIPT``. What would ``word.slice(4).toUpperCase()`` return?  TRY IT!  (`Repl.it <https://repl.it/@launchcode/Intro-To-Method-Chaining>`_).
+
+Check Your Understanding
+------------------------
+
+If you haven't aready, read the method detail pages linked in the table above for the methods ``replace``, ``slice``, and ``trim``. Then answer the following questions.
```

Comment:
Typo: "already" instead of "aready".

Rephrasing suggestion: How about, "Follow the links in the table above for the ``replace``, ``slice``, and ``trim`` methods.  Review the content and then answer the following questions."